test/unit/test_label_procs.py

```diff
@@ -182,3 +183,79 @@ def test_create_label_with_empty_string_name(conn, timestamp_string):
     assert "done" in str(
         run_proc(conn, sql)
     ), "Stored procedure output does not match expected result!"
+
```

Comment:
I think for this test we only really need to validate that the conditions compile. Somethign like:

``` sql
conditions := (select condition from internal.predefined_labels);
for condition in conditions
select condition from query_history where false;
end;
```
the above is pseudo code, we would likely want to report which failed and which passed and not just fail hard on the whole test

Reply:
I added a stored procedure. `INTERNAL.VALIDATE_PREDEFINED_LABELS` to do the check.